src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import './TodoForm.css'
+
+//props = props.addTodo
+class TodoForm extends Component{
+
+    state = {
+        task: '',
+
+    }
+
+
+    changeHandler = (event) =>{
+        this.setState({
+            task: event.target.value
+        })
+    }
+
+    submitHandler = (event) =>{
+        event.preventDefault();
+        //call addTodo of App.js
+        if(this.state.task){
+            this.props.addTodo(this.state.task)
+        }
+
+        this.setState({
+            task: ''
+        })
+
+    }
+
+    clearHandler = () =>{
+        this.props.clearCompleted()
+    }
+
+    render(){
+        return (
+            <div className='formContainer'>
+                <form onSubmit={this.submitHandler}>
+                    <input placeholder="...todo" type="text" onChange={this.changeHandler} value={this.state.task} />
```

Comment:
Can be a good idea to break out multiple props onto newlines for readability.
Example: 
```javascript
<input
        onChange={props.handleTodoChange}
        type="text"
        name="todo"
        value={props.value}
        placeholder="...todo"
      />
```